components/Cart.css

```diff
@@ -73,7 +72,6 @@
 
 .summary {
   margin-top: 12rem;
-  /* margin-left: 2rem; */
   margin-left: -5rem;
```

Comment:
You can use margin shorthand property here.

Reply:
Okayy!! Will do that.